### ISSUE TITLE
[Bug]: The loaded area stops working when you exit the game and enter.

### ISSUE THREAD

**Issue by ExpoTr** [FTBTeam/FTB-Mods-Issues#877]
### Mod

Ftb chunks

### Mod version

Last version for 1.18.2

### Forge / Fabric version

1.18.2 (last version)

### Modpack & version

Create:Astral (last version)

### What issue are you having?

I'm about to stop playing minecraft with mods because of this silly mistake. It's fine if there is a solution to this with the configuration. Otherwise I can't spend any more time on these silly mistakes. As you read in the title. All the loaded fields stop working when I exit the game. It took me 1 hour to find the cause of the error. Do you ever test mods?

### Crashlogs

No.

### Steps to reproduce

load a chunk with automation. then exit the game and re-enter it and go away and check the system. The system will not work unless you are nearby.

### Anything else to note?

No.

**Comment by ExpoTr** [comment]
no, it's not enabled. I just increased the stack load count from 25 to 1,000 in the config. I didn't touch any other settings. but I'll try what you said. i have one question.when I load a chunk does it have a random tick? And does it work between dimensions ? The latest version for the version 1.18.2 of the mod I'm playing.

**Comment by desht** [comment]
> no, it's not enabled

Well, no wonder it's not working, then.  Enable offline chunkloading in your server config, and your problems should be solved.

> when I load a chunk does it have a random tick?

If you mean are blocks (not block entities) randomly ticked, then no.

> And does it work between dimensions ?

Question doesn't make sense.  Does what work between dimensions?
